docs/npm-lerna.md

```diff
@@ -23,7 +23,24 @@ If the version in `package.json` contains one of these patterns, Shipit will pub
 ## Publishing using Lerna
 
 [Lerna](https://github.com/lerna/lerna) is a tool for managing projects with multiple npm packages.
-Shipit understands `lerna.json` files (up to version `2.9.x`), and will publish all packages in a Lerna project to npm.
+Shipit understands `lerna.json` files (up to version `3.22.x`), and will publish all packages in a Lerna project to npm.
+
+### From Git
+
+In addition to the semver keywords supported by [lerna version](https://github.com/lerna/lerna/tree/main/commands/version#positionals), [lerna publish](https://github.com/lerna/lerna/tree/main/commands/publish) also supports the from-git keyword. This will identify packages tagged by lerna version and publish them to npm. This is useful in CI scenarios where you wish to manually increment versions, but have the package contents themselves consistently published by an automated process.
```

Comment:
```suggestion
In addition to the `semver` keyword supported by [`lerna version`](https://github.com/lerna/lerna/tree/main/commands/version#positionals), [`lerna publish`](https://github.com/lerna/lerna/tree/main/commands/publish) also supports the `from-git` keyword. This will identify packages tagged by `lerna version` and publish them to npm. This is useful in CI scenarios where you wish to manually increment versions, but have the package contents themselves consistently published by an automated process.
```